[{"commit":{"message":"Fix the comment"},"files":[{"filename":"test\/jdk\/com\/sun\/jdi\/BadHandshakeTest.java"}],"sha":"b16810b65ed2286231c49c2637e60dc4287a8893"},{"commit":{"message":"8252117: com\/sun\/jdi\/BadHandshakeTest.java failed with \"ConnectException: Connection refused: connect\""},"files":[{"filename":"test\/jdk\/com\/sun\/jdi\/BadHandshakeTest.java"}],"sha":"26c5e4be8dee6e9e5d22849500646c7ab984e1b3"}]